README.md

```diff
@@ -1,2 +1,66 @@
-# wodin
-odin on the web
+# WODIN
+
+[odin](https://github.com/mrc-ide/odin) on the web
+
+## Introduction
+
+WODIN is a web application with [Node.js](https://nodejs.org/en/) back end using [Express](http://expressjs.com/), 
+and [Vue.js](https://vuejs.org/) front end. Both parts are written in [Typescript](https://www.typescriptlang.org/).  
+
+The application is intended to be deployable with custom configuration allowing (typically) teaching courses to be delivered
+where students can author and run model code using [odin](https://github.com/mrc-ide/odin). 
+Odin will be invoked by WODIN to provide executable javascript
+models given odin code written by the users or provided in config. 
```

Comment:
```suggestion
By deploying the application with custom configuration, teaching courses may be created that allow students to author and run model code using [odin](https://github.com/mrc-ide/odin). 
Users can invoke odin to provide executable JavaScript models by writing odin code or providing config to the WODIN interface.
```

Reply:
This bit does need improvement, but I feel that starting the sentence with that "By deploying the application.." clause is not going to be very clear either, and I'm not completely convinced by "Users can invoke odin..". That's what's happening under the hood, but it's not what users are directly interested in. 

How about something like:
"The WODIN application can be deployed with custom configuration to create teaching courses that allow students to author and run model code using odin. 

"When the user runs a model in WODIN, odin will be invoked to provide executable JavaScript for odin model code. This odin code may be written by the user or provided in config."

Reply:
Perfect